supply/lib/supply/uploader.rb

```diff
@@ -106,6 +112,7 @@ def upload_screenshots(language)
 
     def upload_binaries
       apk_paths = [Supply.config[:apk]] unless (apk_paths = Supply.config[:apk_paths])
+      apk_paths.compact!
```

Comment:
why could this be necessary?

Reply:
`[Supply.config[:apk]]` turns into `[nil]` if none are passed in